webhooks/pod_webhook.go

```diff
@@ -319,21 +362,29 @@ func (m *PodMutator) handleFilepathProvider(ctx context.Context, pod *corev1.Pod
 			},
 		},
 	})
+
 	mountPath := fmt.Sprintf("%s/%s", rootFileSyncMountPath, utils.FeatureFlagConfigurationId(ns, n))
 	sidecar.VolumeMounts = append(sidecar.VolumeMounts, corev1.VolumeMount{
 		Name: n,
 		// create a directory mount per featureFlag spec
 		// file mounts will not work
 		MountPath: mountPath,
 	})
-	sidecar.Args = append(
-		sidecar.Args,
-		"--uri",
-		fmt.Sprintf("file:%s/%s",
-			mountPath,
-			utils.FeatureFlagConfigurationConfigMapKey(ns, n),
-		),
-	)
+
+	return types.SourceConfig{
+		URI: fmt.Sprintf("%s/%s", mountPath, utils.FeatureFlagConfigurationConfigMapKey(ns, n)),
+		// todo - this constant needs to be aligned with flagd. We have a mixed usage of file vs filepath
+		Provider: "file",
```

Comment:
:+1:

Reply:
Ceated an issue to track this :) 

https://github.com/open-feature/open-feature-operator/issues/433